### PR TITLE
prepare release v1.16.0

### PR DESCRIPTION
**Pull request by ammolitor**
# Description

prepare release v1.16.0

### CODE REVIEW

**Comment by johscheuer** [comment]
This is the issue in `fdbcli`:

```text
{"level":"debug","ts":1679696158.8820195,"logger":"events","msg":"Normal","object":{"kind":"FoundationDBCluster","namespace":"test-operator-upgrades-716-citvc6m9","name":"fdb-cluster-z38d9g7r","uid":"06fa9114-5146-4aef-9f66-80a311166079","apiVersion":"apps.foundationdb.org/v1beta2","resourceVersion":"694674614"},"reason":"ExcludingProcesses","message":"Excluding [100.80.17.176 100.80.43.192 100.80.54.199 100.80.38.68 100.80.52.147 100.80.18.205 100.80.2.121 100.80.42.48 100.80.32.179 100.80.9.192 100.80.57.244 100.80.56.228 100.80.30.82]"}
{"level":"error","ts":1679696159.1801887,"logger":"fdbclient","msg":"Error from FDB command","namespace":"test-operator-upgrades-716-citvc6m9","cluster":"fdb-cluster-z38d9g7r","code":1,"stdout":"ERROR: This exclude may cause the total free space in the cluster to drop below 10%.\nType `exclude FORCE <ADDRESS...>' to exclude without checking free space.\n","stderr":"","error":"exit status 1","stacktrace":"github.com/FoundationDB/fdb-kubernetes-operator/fdbclient.(*cliAdminClient).runCommandWithBackoff\n\t/workspace/fdbclient/admin_client.go:282\ngithub.com/FoundationDB/fdb-kubernetes-operator/fdbclient.(*cliAdminClient).ExcludeProcesses\n\t/workspace/fdbclient/admin_client.go:432\ngithub.com/FoundationDB/fdb-kubernetes-operator/controllers.excludeProcesses.reconcile\n\t/workspace/controllers/exclude_processes.go:84\ngithub.com/FoundationDB/fdb-kubernetes-operator/controllers.(*FoundationDBClusterReconciler).Reconcile\n\t/workspace/controllers/cluster_controller.go:166\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Reconcile\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.3/pkg/internal/controller/controller.go:121\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.3/pkg/internal/controller/controller.go:320\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.3/pkg/internal/controller/controller.go:273\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2\n\t/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.3/pkg/internal/controller/controller.go:234"}
```

We are tracking that issue separately and we have a fix for fdbcli in the next patch release of 7.1

File: docs/changelog/v1.16.0.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# v1.16.0
+
+## Changes
+
+### Operator
+
+Ensure we call exclude only on missing processes  bug  Something isn't working technical-debt [#1550](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1550)
```
Comment (johscheuer) [inline docs/changelog/v1.16.0.md:7]:
```suggestion
Ensure we call exclude only on missing processes [#1550](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1550)
```

was it intended to include those labels?

File: docs/changelog/v1.16.0.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Make sure we always check the sidecar to get the most recent information about the updated ConfigMap bug Something isn't working [#1547](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1547)
```
Comment (johscheuer) [inline docs/changelog/v1.16.0.md:8]:
```suggestion
Make sure we always check the sidecar to get the most recent information about the updated ConfigMap [#1547](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1547)
```

File: docs/changelog/v1.16.0.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+Initial design for replacing Pods running on tainted nodes [#1502](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1502)
+Update examples and better document how to run a multi-region FDB cluster  documentation Improvements or additions to documentation [#1548](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1548)
```
Comment (johscheuer) [inline docs/changelog/v1.16.0.md:10]:
```suggestion
Update examples and better document how to run a multi-region FDB cluster [#1548](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1548)
```

File: docs/changelog/v1.16.0.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+Add kube builder validation checks for a couple of entries.  [#1544](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1544)
+Adding a way to ignore specific LogGroups in check client compatibility [#1526](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1526)
+Fix upgrade logic for multiple storage server per Pod  bug Something isn't working [#1538](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1538)
```
Comment (johscheuer) [inline docs/changelog/v1.16.0.md:13]:
```suggestion
Fix upgrade logic for multiple storage server per Pod [#1538](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1538)
```

**Review by johscheuer — CHANGES_REQUESTED** [review]
Seems like the script is not working properly with PRs that have one or more labels

File: docs/changelog/v1.16.0.md (hunk 1 above)
Comment (ammolitor) [reply docs/changelog/v1.16.0.md:7]: there is no script for this project. this is manually created.

**Pull request merged**